drivers/hmis/app/graphql/types/hmis_schema/funder_input.rb

```diff
@@ -0,0 +1,21 @@
+module Types
+  class HmisSchema::FunderInput < BaseInputObject
+    def self.source_type
+      Types::HmisSchema::Funder
+    end
+
+    hud_argument :project_id, ID, required: false
+    hud_argument :funder, HmisSchema::Enums::FundingSource
+    hud_argument :other_funder
+    hud_argument :grant_id
+    hud_argument :start_date
+    hud_argument :end_date
```

Comment:
@eanders I started feeling like I was really going to regret what I did 😬  sooo I just implemented it as a custom field `hud_field` and argument `hud_argument` instead. it infers the type and nullability. this has a less imposing structure for now, and is more of an optional convenience.

Reply:
also wanted to get Zach's input before doing something drastic

Reply:
Ok.  I think collecting everything up in a method or constant will be beneficial later, but that's easy enough to add back, and this moves a bunch of the repetition off to one location, so that's a win.